### PR TITLE
Upload multiple private PGP keys on a passphrase change.

### LINKED ISSUE #1: Support multiple Private keys in uploads
Right now we only support 1.  This should be doable now that keybase/keybase#338 has done the server-side.

### PR DESCRIPTION
**Pull request by maxtaco**
Closes #1.

### CODE REVIEW

**Comment by maxtaco** [comment]
r @malgorithms ?

File: src/hilevel/account.iced [hunk 1]
```diff
@@ -247,15 +247,16 @@ exports.Account = class Account
 
   #---------------
 
-  _cpp2_reencrypt_pgp_private_key : ( { me, old_ppc, new_ppc }, cb ) ->
-    output = null
+  _cpp2_reencrypt_pgp_private_keys : ( { me, old_ppc, new_ppc }, cb ) ->
```
Comment (malgorithms) [inline src/hilevel/account.iced]: what is cpp2?

**Comment by malgorithms** [comment]
if this passes tests (and cpp2 gets a clearer name) then it's good for me.

File: src/hilevel/account.iced [hunk 2]
```diff
@@ -247,15 +247,16 @@ exports.Account = class Account
...
+    outputs = []
     esc = make_esc cb, "_cpp2_reencrypt_pgp_private_key"
-    if (key = me?.private_keys?.primary?.bundle)?
-      await KeyManager.import_from_p3skb { armored : key }, esc defer km
+    for {bundle} in (me?.private_keys?.all or [])
+      await KeyManager.import_from_p3skb { armored : bundle }, esc defer km
```
Comment (maxtaco) [inline src/hilevel/account.iced]: This actually won't work, since we'll be resetting the TripleSec via scrub every time we unlock.

**Comment by maxtaco** [comment]
need tests.

**Comment by maxtaco** [comment]
Added tests in keybase/keybase.  Changed method names as requested.

**Pull request merged**